agent/utils/files/file_op.go

```diff
@@ -293,20 +289,14 @@ func (f FileOp) DownloadFileWithProcess(url, dst, key string, ignoreCertificate
 		out.Close()
 		resp.Body.Close()
 
-		value, err := global.CACHE.Get(counter.Key)
-		if err != nil {
-			global.LOG.Errorf("get cache error,err %s", err.Error())
-			return
-		}
+		value := global.CACHE.Get(counter.Key)
 		process := &Process{}
-		_ = json.Unmarshal(value, process)
+		_ = json.Unmarshal([]byte(value), process)
 		process.Percent = 100
 		process.Name = counter.Name
 		process.Total = process.Written
 		by, _ := json.Marshal(process)
-		if err := global.CACHE.SetWithTTL(counter.Key, string(by), time.Second*time.Duration(10)); err != nil {
-			global.LOG.Errorf("save cache error, err %s", err.Error())
-		}
+		global.CACHE.Set(counter.Key, string(by))
 	}()
 	return nil
 }
```

Comment:
The code seems to be a Go program with some errors, such as unnecessary curly braces that should have been removed or merged into previous blocks. Additionally, there are no references to JSON serialization which makes it difficult to analyze its logic effectively. I would not recommend this codebase unless corrections are made and fully debugged by the programmer who originally wrote this snippet.

For instance, `global.CACHE` could reference an incorrect package name since it does not follow standard practice for caching systems used by Go. To avoid future similar cases of inconsistent coding style (e.g., removing unused packages), consider using precompiled templates like GOROOT/gofmt for better quality control during compilation. The comments at specific points can give more context but the actual implementation structure is lacking clarity regarding what's going on here.